clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -1190,3 +1192,65 @@ export const CustomCheckbox = ({
     </FormControl>
   );
 };
+
+interface CustomClipboardCopyProps {
+  label?: string;
+  tooltip?: string;
+  variant?: Variant;
+}
+
+export const CustomClipboardCopy = ({
```

Comment:
New form field component for the clipboard copy functionality
<img width="743" alt="image" src="https://github.com/ethyca/fides/assets/2092272/f120fd2d-1728-45d2-bcfa-fc2cd59cfa8c">